sympy/physics/control/lti.py

```diff
@@ -1092,6 +1092,16 @@ def is_stable(self):
         return fuzzy_and(pole.as_real_imag()[0].is_negative for pole in self.poles())
 
     def __add__(self, other):
+        if isinstance(other, Feedback):
+            arg_list = list(other.sys1.args) if isinstance(other.sys1, Series) else [other.sys1]
```

Comment:
Try to make lines like these simpler and more readable. It will be helpful to future users.

Reply:
I've changed these lines to a simple if else statement in the updated commit.